### PR TITLE
Cleaning requirements.txt file and putting readme for a quick start.

### PR DESCRIPTION
**Pull request by yashkumaratri**
Cleaned unused imports to conserve time while installing and created README.md to help quick start the code without going through the code and finding the way to run it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# ActiveLearning
+MNIST Active Learning
+
+Requirements to install ( python3.5 or above )
```
Comment (smatt989) [inline README.md]: is this true?  pretty sure it is 2.7?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+`python3 activemnist.py 1 10 2000 True old True 5`
+
+Additional Running Info
+
+######runs a test with the following parameters:
+
+###1st parameter: number of runs (almost always want just 1 or will be extremely long)
+
+###2nd parameter: how bit each mini-batch should be
```
Comment (smatt989) [inline README.md]: *big

**Comment by smatt989** [comment]
thanks for doing this!  take a look at my 2 comments and then i'll merge!

File: README.md (hunk 1 above)
Comment (yashkumaratri) [reply README.md]: Tested with Py2, Works like charm.

File: README.md (hunk 2 above)
Comment (yashkumaratri) [reply README.md]: Corrected.

**Pull request merged**